src/components/Calculator.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+const Buttons = () => (
+  <>
+    <li className="gray-button">AC</li>
+    <li className="gray-button">+/-</li>
+    <li className="gray-button">%</li>
+    <li className="orange-button">÷</li>
+    <li className="gray-button">7</li>
+    <li className="gray-button">8</li>
+    <li className="gray-button">9</li>
+    <li className="orange-button">x</li>
+    <li className="gray-button">4</li>
+    <li className="gray-button">5</li>
+    <li className="gray-button">6</li>
+    <li className="orange-button">-</li>
+    <li className="gray-button">1</li>
+    <li className="gray-button">2</li>
+    <li className="gray-button">3</li>
+    <li className="orange-button">+</li>
+    <li className="gray-button large">0</li>
+    <li className="gray-button">.</li>
+    <li className="orange-button">=</li>
+  </>
+);
+
+export default class Calculator extends React.PureComponent {
```

Comment:
Please be sure to have an App.js (main component) file that is class-based as it is part of the requirements 🤓 Kindly import your `Calculator component` to the App component 📝